test/kwtool_web/controllers/api/v1/search_controller_test.exs

```diff
@@ -0,0 +1,220 @@
+defmodule KwtoolWeb.Api.V1.SearchControllerTest do
+  use KwtoolWeb.ConnCase, async: true
+
+  import KwtoolWeb.Support.ConnHelper
+
+  describe "get index/2" do
+    test "given search with keyword param, returns a list of matching keyword", %{
+      conn: conn
+    } do
+      created_user = insert(:user)
+      keyword = insert(:keyword, phrase: "cheap flights", status: "finished", user: created_user)
+      insert(:keyword_result, keyword: keyword)
+
+      conn =
+        conn
+        |> with_signed_in_user(created_user)
+        |> get(Routes.api_search_path(conn, :index, %{"keyword" => "cheap flights"}))
+
+      assert %{
+               "data" => [
+                 %{
+                   "attributes" => %{
+                     "inserted_at" => _,
+                     "phrase" => "cheap flights",
+                     "status" => "finished",
+                     "updated_at" => _
+                   },
+                   "id" => _,
+                   "relationships" => %{
+                     "keyword_results" => %{"data" => [%{"id" => _, "type" => "keyword_result"}]}
+                   },
+                   "type" => "keywords"
+                 }
+               ],
+               "included" => [
+                 %{
+                   "attributes" => %{
+                     "all_ads_count" => 2,
+                     "all_links_count" => 2,
+                     "organic_result_count" => 2,
+                     "organic_result_urls" => ["https://company-a.com", "https://company-b.com"],
+                     "top_ads_count" => 2,
+                     "top_ads_urls" => ["https://ads-product-a.com", "https://ads-product-b.com"]
+                   },
+                   "id" => _,
+                   "relationships" => %{},
+                   "type" => "keyword_result"
+                 }
+               ]
+             } = json_response(conn, 200)
+    end
+
+    test "given search with keyword, url, and min ads count params, returns keyword that matching given filters",
```

Comment:
```suggestion
    test "given search with keyword, url, and min ads count params, returns keywords that matching given filters",
```

Reply:
Updated in https://github.com/byhbt/kwtool/pull/49/commits/634ec9ecd477dff893d4f82ebe2a8ab75e064701